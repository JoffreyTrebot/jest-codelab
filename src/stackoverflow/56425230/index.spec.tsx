import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Header } from './';

describe('Header', () => {
  it('match snapshot', () => {
    jest.spyOn(Date, 'now').mockReturnValueOnce(new Date('2019/11/24').getTime());
    const wrapper: ShallowWrapper = shallow(<Header></Header>);
    expect(wrapper.text()).toBe(new Date('2019/11/24').toISOString());
    expect(wrapper).toMatchSnapshot();
  });
});
