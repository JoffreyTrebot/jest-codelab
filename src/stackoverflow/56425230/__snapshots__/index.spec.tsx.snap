// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header match snapshot 1`] = `
<div>
  Tue Dec 24 2019 00:00:00 GMT+0800
</div>
`;
